src/Microsoft.Extensions.CodeGenerators.Mvc/Controller/MvcControllerEmpty.cs

```diff
@@ -13,13 +13,33 @@ namespace Microsoft.Extensions.CodeGenerators.Mvc.Controller
     public class MvcControllerEmpty : ControllerGeneratorBase
     {
         public MvcControllerEmpty(
-            [NotNull]ILibraryManager libraryManager,
-            [NotNull]IApplicationEnvironment environment,
-            [NotNull]ICodeGeneratorActionsService codeGeneratorActionsService,
-            [NotNull]IServiceProvider serviceProvider,
-            [NotNull]ILogger logger)
+            ILibraryManager libraryManager,
+            IApplicationEnvironment environment,
+            ICodeGeneratorActionsService codeGeneratorActionsService,
+            IServiceProvider serviceProvider,
+            ILogger logger)
             : base(libraryManager, environment, codeGeneratorActionsService, serviceProvider, logger)
         {
+            if (libraryManager == null)
+            {
+                throw new ArgumentNullException(nameof(libraryManager));
+            }
+            if (environment == null)
```

Comment:
just to be consistent with everything else you've done here separate each of em by empty lines.